README.md

```diff
@@ -31,6 +31,8 @@ Additionally, you can find details and examples of how our API is structured in
 
 The API Reference Guide provides examples of what information is needed for a particular request and how that information would be formatted. Using those examples, you can easily determine what methods would be necessary to include that information in a request using this SDK.
 
+## MetaKey Support
```

Comment:
Replace this content with what is written in Client SDK for Java.

Also, content for OAuth is missing.

Reply:
Done